server/example.env

```diff
@@ -22,7 +22,6 @@ TWIML_APP_SID=APxxxxxxxxxxxxxx
 CALLER_ID=+11234567890
 
 # Push credential for incoming calls
-# Please see https://github.com/twilio/twilio-voice-react-native#incoming-calls
```

Comment:
Instead of removing, do we have an updated link that provides more information about incoming calls, push credential sid, etc?

Reply:
As someone who attempted to use this, I could not finish this without the documentation that was supposed to be at the link. If someone like me should be able to use this in the future, this is definitely important 👍